### PR TITLE
Remove Notification model and rename database table and columns for n…

### PR DESCRIPTION
**Pull request by clark800**
…otifications

### CODE REVIEW

File: src/lib/notificationWorker.js [hunk 1]
```diff
@@ -18,35 +17,40 @@ const convertToExternalTransfer = require('../models/converters/transfers')
   .convertToExternalTransfer
 const getAffectedSubscriptions = require('../models/db/subscriptions')
   .getAffectedSubscriptions
+const getMatchingNotification = require('../models/db/notifications')
+  .getMatchingNotification
+const notificationDAO = require('../models/db/notifications')
```
Comment (lumberj) [inline src/lib/notificationWorker.js]: What is DAO?

Comment (clark800) [inline src/lib/notificationWorker.js]: https://en.wikipedia.org/wiki/Data_access_object

Comment (clark800) [inline src/lib/notificationWorker.js]: Not to be confused with "The DAO" https://daohub.org/ or "Dao" https://en.wikipedia.org/wiki/Tao or ... :)

**Comment by lumberj** [comment]
LGTM

**Pull request merged**